content/en/docs/reference/kubernetes-api/ref/common-definitions/quantity-.md

```diff
@@ -0,0 +1,57 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/apimachinery/pkg/api/resource"
+  kind: "Quantity"
+collapsible: false
+content_type: "api_reference"
+description: "Quantity is a fixed-point representation of a number."
+draft: false
+title: "Quantity"
+weight: 18
+---
+
+
+
+`import "k8s.io/apimachinery/pkg/api/resource"`
+
+
+## Quantity
+
+Quantity is a fixed-point representation of a number. It provides convenient marshaling/unmarshaling in JSON and YAML, in addition to String() and AsInt64() accessors.
+
+The serialization format is:
+
+<quantity>        ::= <signedNumber><suffix>
+  (Note that <suffix> may be empty, from the "" case in <decimalSI>.)
+<digit>           ::= 0 | 1 | ... | 9 <digits>          ::= <digit> | <digit><digits> <number>          ::= <digits> | <digits>.<digits> | <digits>. | .<digits> <sign>            ::= "+" | "-" <signedNumber>    ::= <number> | <sign><number> <suffix>          ::= <binarySI> | <decimalExponent> | <decimalSI> <binarySI>        ::= Ki | Mi | Gi | Ti | Pi | Ei
+  (International System of units; See: http://physics.nist.gov/cuu/Units/binary.html)
+<decimalSI>       ::= m | "" | k | M | G | T | P | E
+  (Note that 1024 = 1Ki but 1000 = 1k; I didn't choose the capitalization.)
+<decimalExponent> ::= "e" <signedNumber> | "E" <signedNumber>
```

Comment:
The above output would be terrible. All `<fooBar>` will be interpreted as HTML tags and thus masked out.

Reply:
Sure, this description is not easy to output in a fancy way.

As there is no norm for the format of definition description in the Go sources, we cannot rely on a specific format and apply a specific transformation. Probably some normalization would help.

Reply:
Maybe
```markdown
\<decimalExponent> ::= "e" \<signedNumber> | "E" \<signedNumber> 
```
?

Reply:
The generator is now escaping < with \\<. That seems ok to me